work_items/kyc-vcs/requirements-to-accept-a-kyc-vc.md

```diff
@@ -0,0 +1,590 @@
+2. Discover information through the Issuer’s DID or DID website, or
+3. Decide to trust the VC based on the contents and signature proof meeting their requirements.
+
+**<span style="text-decoration:underline;">Example 2: Resolve Issuer DID</span>**
+
+[Resolution of the Issuer's DID](https://w3c-ccg.github.io/did-resolution/) yields the result:
+
+```json
+{
+   "@context": [
+      "https://www.w3.org/ns/did/v1",
+      "https://w3id.org/security/suites/ed25519-2020/v1"
+  ],
+   "id": "did:example:BlockInc",
+   "authentication": [{
+      "id": "did:example:BlockInc#keys-1",
+      "type": "Ed25519VerificationKey2020",
+      "controller": "did:example:BlockInc",
+      "publicKeyMultibase": "zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+   }],
+   "serviceEndpoint" : {
+      "id": "did:example:BlockInc#dwn-1" ,
+      "type": "DWN",
+      "serviceEndpoint": "https://example.com/my-dwn" 
+   }
+}
+```
+
+### 2. CIP/KYC Vocabulary
+
+The KYC Credential for a global customer will include some or all of the following data fields:
+
+<table>
+  <tr>
+   <td><strong>Vocabulary</strong>
+   </td>
+   <td><strong>Field Description & References</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Family Name 
+   </td>
+   <td>Refer to <a href="https://schema.org/Person">Person Schema</a>
+<p>
+In the USA, the Family Name is the <strong>Last Name</strong> of a Person
+   </td>
+  </tr>
+  <tr>
+   <td>Given Name
+   </td>
+   <td>Refer to <a href="https://schema.org/Person">Person Schema</a>
+<p>
+In the USA, the Given Name is the <strong>First Name</strong> of a Person
+   </td>
+  </tr>
+  <tr>
+   <td>Additional Name
+   </td>
+   <td>Refer to <a href="https://schema.org/Person">Person Schema</a>
+<p>
+This can be used to include <strong>Middle Name</strong> or left blank or n/a
+   </td>
+  </tr>
+  <tr>
+   <td>Birth Date (DOB)
+   </td>
+   <td>Universal Date Format (<a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO 8601</a>): YYYY/MM/DD
+   </td>
+  </tr>
+  <tr>
+   <td>Postal Address 
+   </td>
+   <td>Refer to <a href="https://schema.org/PostalAddress">Postal Address Schema</a>
+<p>
+Data fields should include: 
+<p>
+“addressCountry” - Country using the <a href="https://www.iso.org/iso-3166-country-codes.html">ISO</a> 3 Alpha Character country abbreviation
+<p>
+“addressLocality” - City or locality within a region
+<p>
+“addressRegion” - County, Province or State (US)
+<p>
+“postOfficeBoxNumber” - not a valid address for financial transactions in the US
+<p>
+“postalCode” - Zip Code in the USA
+<p>
+“streetAddress” - House number and Street name for example “601 Church Street”
+   </td>
+  </tr>
+  <tr>
+   <td>TaxID
+   </td>
+   <td>Refer to <a href="https://schema.org/Person">Person Schema</a>
+<p>
+“123-45-6789” This can include an SSN or ITIN (in the US), National ID number (in Canada), National ID Card number, or other government issued identity number. 
```

Comment:
```suggestion
“123-45-6789” This can include an SSN or ITIN (in the US), National ID number (in Canada), or other government issued identity number. 
```